infra/scripts/trigger-github-actions-pr-workflow.sh

```diff
@@ -0,0 +1,16 @@
+# Trigger Github Actions PR Workflow by creating a workflow dispatch event via a POST request.
+# Expects environment variables to be passed:
+# PULL_PULL_SHA - pull request hea SHA.
+# GCP_PROJECT_ID - GCP Project ID to pass to the PR workflow.
+# GCP_SA_KEY - Service Account credentials in JSON format to pass to the PR workflow.
+curl -X POST \
```

Comment:
By the way it may be easier to use `base64` encoding when sending the JSON blob. Not sure if you are running into any transmission issues. Should be easy to unencode on the other side.

Reply:
This is a good point and I did encounter issues encoding JSON as string.
However I've change the approached used completely, so this is not longer relevant.